### PR TITLE
Add Priority levels

### LINKED ISSUE #83: Add priority levels to events

### LINKED ISSUE #84: Add "Sort" command to sort events by priority level

### LINKED ISSUE #85: Add support for "sort" command to parser

### LINKED ISSUE #90: As an event organiser, I want my events to have different priority levels, so that I can easily track which events are most important or urgent
- #83 
- #84 
- #85 
- #86 
- #87

### PR DESCRIPTION
**Pull request by LTK-1606**
Add enumeration class for Priority levels
Update all existing methods  to include priority levels
Add functionality to sort by priority levels
Update parser to support
Modify JUnit tests

Fixes #83 
Fixes #84 
Fixes #85 
Fixes #90

### CODE REVIEW

File: text-ui-test/EXPECTED.TXT [hunk 1]
```diff
@@ -46,16 +46,16 @@ Enter a command: There are 1 participants in dinner party! Here are your partici
 ------------------------
 Enter a command: Event added successfully
 ------------------------
-Enter a command: name
+Enter a command:
 Events successfully sorted by name!
```
Comment (jemehgoh) [inline text-ui-test/EXPECTED.TXT:50]: I think you have to remove the newline here (i.e. merge lines 49-50 into one line and do the same for lines 55-56).

File: src/main/java/seedu/manager/enumeration/Priority.java [hunk 2]
```diff
@@ -0,0 +1,5 @@
+package seedu.manager.enumeration;
+
+public enum Priority {
+    HIGH, MEDIUM, LOW
+}
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/enumeration/Priority.java:5]: I like how you used an enumeration to store the priorities!

File: src/main/java/seedu/manager/event/Event.java [hunk 3]
```diff
@@ -44,6 +46,24 @@ public Event(String eventName, LocalDateTime eventTime, String eventVenue) {
...
     }
 
+    //@@author LTK-1606
+    /**
+     * Constructs an Event with the specified name, time, venue and priority.
+     *
+     * @param eventName  the name of the event
+     * @param eventTime  the time duration of the event
+     * @param eventVenue the venue of the event
+     * @param eventPriority the priority level of the event
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/event/Event.java:56]: I think you could add punctuation for the param descriptions here?

File: src/main/java/seedu/manager/command/SortCommand.java [hunk 4]
```diff
@@ -11,6 +11,8 @@ public class SortCommand extends Command {
 
     private static final String SORT_BY_NAME_MESSAGE = "Events successfully sorted by name!";
     private static final String SORT_BY_TIME_MESSAGE = "Events successfully sorted by time!";
+    private static final String SORT_BY_PRIORITY_MESSAGE = "Events successfully sorted by priority level!";
+
```
Comment (jemehgoh) [inline src/main/java/seedu/manager/command/SortCommand.java:15]: I think this additional blank line can be removed?

**Review by jemehgoh — COMMENTED** [review]
Mostly LGTM, just a few nits.

**Comment by LTK-1606** [comment]
Priority, not PRIORITY

**Pull request merged**